src/main/java/crazypants/enderio/machine/soul/BlockSoulBinder.java

```diff
@@ -35,7 +34,7 @@
 
   public static BlockSoulBinder create() {
     PacketDrainPlayerXP.register();
-    PacketExperienceContainer.register();
+    PacketExperienceContainer.register(); //TODO Fix
```

Comment:
I think I moved all packet registration in base into the Network class. We should do that in machines, too.